ui-admin/src/study/StudyContent.tsx

```diff
@@ -116,66 +111,52 @@ function StudyContent({ studyEnvContext }: {studyEnvContext: StudyEnvContextT})
               </ul>
             </div>
           </li>
-          <li className="mb-3 bg-white">
-            <div style={contentHeaderStyle}>
-              <h6>Surveys</h6>
+          <li className="mb-3 rounded-2 p-3" style={{ background: '#efefef' }}>
+            <h6>Research Surveys</h6>
+            <div className="flex-grow-1 pt-3">
+              <ul className="list-unstyled">
+                { configuredResearchSurveys.map((surveyConfig, index) => {
+                  const survey = surveyConfig.survey
+                  return <SurveyListItem key={index} survey={survey} surveyConfig={surveyConfig}
+                    setSelectedSurveyConfig={setSelectedSurveyConfig}
+                    updateConfiguredSurvey={updateConfiguredSurvey}
+                    setShowDeleteSurveyModal={setShowDeleteSurveyModal}
+                    setShowArchiveSurveyModal={setShowArchiveSurveyModal}
+                    showArchiveSurveyModal={showArchiveSurveyModal}
+                    showDeleteSurveyModal={showDeleteSurveyModal}
+                    isReadOnlyEnv={isReadOnlyEnv}/>
+                })}
+                {!isReadOnlyEnv && <li>
+                  <Button variant="secondary" data-testid={'addResearchSurvey'} onClick={() => {
+                    setShowCreateSurveyModal(!showCreateSurveyModal)
+                    setCreateSurveyType('RESEARCH')
+                  }}>
+                    <FontAwesomeIcon icon={faPlus}/> Add
+                  </Button>
+                </li> }
+              </ul>
             </div>
+          </li>
+          <li className="mb-3 rounded-2 p-3" style={{ background: '#efefef' }}>
+            <h6>Outreach</h6>
```

Comment:
As in the ptp detail page, `Outreach` is always listed, even if the study does not have an outreach concept.